views/spaces/index.erb

```diff
@@ -1 +1,24 @@
-<h1>Book a space</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Book A Space</title>
+</head>
+<body>
+<h1>Book A Space</h1>
+<form action="/spaces/new" method="get">
+  <input type="submit" name="list_of_spaces" class="list_a_space" value="List a Space"/>
+</form>
+<div>
+  <ul class="indiv_space">
```

Comment:
is the class still a list of spaces?

Reply:
No, as this is the class for the 'list a space' button